.github/workflows/storybook-tests.yml

```diff
@@ -0,0 +1,29 @@
+name: Storybook Tests
```

Comment:
since we have so many types of tests, can we call this Accessibility tests? or something more appropriate?